themes/vue/source/js/theme-data.js

```diff
@@ -124,6 +124,62 @@ var themeData = [
         price: 0
       }
     ]
+  },
+  {
+    name: 'PrimeVue',
+    description: `The open-source UI component library [PrimeVue](https://www.primefaces.org/primevue/#/?af_id=4218) offers over 50 flexible components to build your apps with! They have a ton of different component themes and Vue-CLI application templates available to get the look&feel that suits you best.`,
+    seeMoreUrl: 'https://www.primefaces.org/primevue/#/?af_id=4218',
+    products: [
+      {
+        name: 'Sapphire',
+        price: 79,
+        description: 'Materal Design Admin Template',
```

Comment:
Is "Materal" intentional, or should it be "Material"?

Reply:
No it is not. I created a PR to fix this. https://github.com/vuejs/vuejs.org/pull/2467